.openshift/rocket-chat-ephemeral.json

```diff
@@ -262,6 +262,30 @@
                                         "mountPath": "/app/uploads"
                                     }
                                 ],
+                                "readinessProbe": {
+                                    "httpGet": {
+                                        "path": "/api/v1/info",
+                                        "port": 3000,
+                                        "scheme": "HTTP"
+                                    },
+                                    "initialDelaySeconds": 5,
+                                    "timeoutSeconds": 1,
+                                    "periodSeconds": 10,
+                                    "successThreshold": 1,
+                                    "failureThreshold": 3
```

Comment:
So on larger servers it takes longer for instances to start up.  Might need to add some variation in here.

I'm actually struggling with this a bit my self. Our demo server takes 2-3 minutes on initial load because of all of the users and channels it has to load into cache.

when I threw our demo server on k8s I had to actually adjust the readiness check to something like:
```
                        "livenessProbe": {
                            "failureThreshold": 3,
                            "httpGet": {
                                "path": "/api/info",
                                "port": 3000,
                                "scheme": "HTTP"
                            },
                            "initialDelaySeconds": 300,
                            "periodSeconds": 10,
                            "successThreshold": 1,
                            "timeoutSeconds": 1
                        },
                        "readinessProbe": {
                            "failureThreshold": 32,
                            "httpGet": {
                                "path": "/api/info",
                                "port": 3000,
                                "scheme": "HTTP"
                            },
                            "initialDelaySeconds": 10,
                            "periodSeconds": 10,
                            "successThreshold": 1,
                            "timeoutSeconds": 1
                        },
```

My thoughts here were readiness probe needs to be agressive.  We want to serve traffic to the instance as fast as possible.  But pushing the liveness check back a bit to keep it from killing an instance that is still in the process of starting up.

What are your thoughts here?